### DISCUSSION TITLE
Better start

### DISCUSSION THREAD

**Discussion by jxnu-liguobin** [kobylynskyi/graphql-java-codegen#552] [General]
I found out when I was learning the rust graphql client. [cynic](https://github.com/obmarg/cynic) provides an interesting and useful function https://generator.cynic-rs.dev/. 
I wonder if our Java can consider something similar? However, we have a lot of configuration items, which may be a troublesome thing. What we can do is show the model, resolvers and the real query of the current query splicing

**Comment by kobylynskyi** [comment]
Wow, I find it a very useful thing for generating a GraphQL client code!

And you are right, we can think of something similar for GraphQL **Java** (+Kotlin, +Scala) Client Code.

**Reply by jxnu-liguobin** [reply]
👍
